muikku/src/main/java/fi/otavanopisto/muikku/session/SessionBackingBean.java

```diff
@@ -321,6 +326,35 @@ private String resolveLoggedUserRoleText() {
   public String getAreaPermissions() {
     return areaPermissions != null ? areaPermissions : "null";
   }
+  
+  public Boolean getUserPropertyAsBoolean(String propertyKey) {
+	String userProperty = this.getUserProperty(propertyKey);
+    if(userProperty != null && "1".equals(userProperty)) {
```

Comment:
Mixed tabs and spaces. Also
`return userProperty != null && "1".equals(userProperty);`
:P